Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -613,9 +520,11 @@ public protocol PurchasesType: AnyObject {
     func checkTrialOrIntroDiscountEligibility(product: StoreProduct) async -> IntroEligibilityStatus
 
     /**
-     * Use this method to fetch ``PromotionalOffer``
-     *  to use in ``purchase(package:promotionalOffer:)`` or ``purchase(product:promotionalOffer:)``.
+     * Use this method to fetch ``PromotionalOffer`` to use in ``purchase(package:promotionalOffer:completion:)``
```

Comment:
This was producing a warning now because they're in the other `protocol`, but it made more sense to reference the completion block methods instead anyway.